powersimdata/scenario/analyze.py

```diff
@@ -226,6 +228,33 @@ def get_load_shed(self):
 
         return load_shed
 
+        def get_load_shift_up(self):
```

Comment:
FYI based on #592 this method would change to just be `return self._get_data("LOAD_SHIFT_UP")`. Similar for `get_load_shift_dn`.

Reply:
Done.